src/gsUtils/gsMesh/gsVertex.h

```diff
@@ -14,7 +14,7 @@
 #pragma once
 
 #include <gsUtils/gsMesh/gsMeshElement.h>
-#include <gsCore/gsLinearAlgebra.h>
+//#include <gsCore/gsLinearAlgebra.h>
```

Comment:
I think we should not comment such things out but just remove them. The same holds for the things below.